FSW/src/util/comUtil.cpp

```diff
@@ -0,0 +1,268 @@
+/* comUtil.cpp handles NS2 payload commands
+ * Usage:
+ *  Reads incoming commands from serial, adds them to the command queue
+ *  and executes each command. 
+ *      
+ * 
+ * Modules encompassed:
+ *  Command Handling
+ *
+ * Additional files needed for compilation:
+ *  config.hpp
+ *  comUtil.hpp
+ *  
+ */
+
+/* - - - - - - Includes - - - - - - */
+// C++ libraries
+
+// Other libraries
+
+// NS2 headers
+#include "../headers/config.hpp"
+#include "../headers/comUtil.hpp"
+
+
+/* Module Variable Definitions */
+int commandQueue[COMMAND_QUEUE_SIZE] = {};  // initializes as all zeros
+static int queueIndex = 0;                  // next available index of command queue
+static bool isPaused = false;               // indicates if command execution is paused
+
+// TODO: set Serial.setTimeout() in config.hpp
```

Comment:
Is this TODO waiting on another feature?

Reply:
Nope, good catch. This will actually go in the initialization function in main. Removed.